package.json

```diff
@@ -43,19 +43,19 @@
     "ui-router-rx": "=0.2.1"
   },
   "peerDependencies": {
-    "@angular/common": "^2.3.1",
-    "@angular/core": "^2.3.1",
+    "@angular/common": "^4.0.0",
+    "@angular/core": "^4.0.0",
     "@angular/router": "^3.3.1"
   },
   "devDependencies": {
-    "@angular/common": "^2.3.1",
-    "@angular/compiler": "^2.3.1",
-    "@angular/compiler-cli": "^2.3.1",
-    "@angular/core": "^2.3.1",
-    "@angular/platform-browser": "^2.3.1",
-    "@angular/platform-browser-dynamic": "^2.3.1",
-    "@angular/platform-server": "^2.3.1",
-    "@angular/router": "^3.3.1",
+    "@angular/common": "^4.0.0",
+    "@angular/compiler": "^4.0.0",
+    "@angular/compiler-cli": "^4.0.0",
+    "@angular/core": "^4.0.0",
+    "@angular/platform-browser": "^4.0.0",
+    "@angular/platform-browser-dynamic": "^4.0.0",
+    "@angular/platform-server": "^4.0.0",
+    "@angular/router": "^4.0.0",
```

Comment:
@aitboudad we'll need to use something along the lines of `>=4.0.0-beta <5.0.0` for the semver constraint here (which is what `@angular/cli` is using by default for projects generated with the `--ng4` flag).

`^4.0.0` does not capture betas or RCs.